cmd/operator-sdk/build/cmd.go

```diff
@@ -106,7 +105,7 @@ func buildFunc(cmd *cobra.Command, args []string) error {
 	image := args[0]
 	projutil.MustInProjectRoot()
 
-	if kbutil.HasProjectFile() {
+	if projutil.HasProjectFile() {
```

Comment:
I saw it after to do https://github.com/operator-framework/operator-sdk/pull/3539 but shows that you and I was with the same ideas :-)